modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/suggestions/spi/SXPBlueprintSuggestionsContributor.java

```diff
@@ -211,6 +211,10 @@ private SearchRequest _getSearchRequest(
 					SearchContextAttributes.ATTRIBUTE_KEY_EMPTY_SEARCH,
 					searchContext1.getAttribute(
 						SearchContextAttributes.ATTRIBUTE_KEY_EMPTY_SEARCH));
+				searchContext2.setAttribute(
```

Comment:
This doesn't seem necessary, as it always seems to be called through [SuggestionResourceImpl.postSuggestionsPage](https://github.com/liferay-search/liferay-portal/pull/1485/commits/73afad5f08be45cc3c8cc51d14a36517ac3afcfe#diff-7bf4ee8557e1ad2318ae6ce78f2f8cabc844591a6b973a2a1cfc05ab79558941)

Reply:
But if there's a different logic path that doesn't set the new variable but calls this method, then it should stay.